src/imars3d/backend/corrections/ring_removal.py

```diff
@@ -6,13 +6,126 @@
 from imars3d.backend.util.functions import clamp_max_workers
 import scipy
 import numpy as np
+
+try:
+    import bm3d_streak_removal as bm3dsr
+except ImportError:
+    # add a log?
+    bm3dsr = None
 from multiprocessing.managers import SharedMemoryManager
 from tqdm.contrib.concurrent import process_map
 from functools import partial
 
 logger = logging.getLogger(__name__)
 
 
+class bm3d_ring_removal(param.ParameterizedFunction):
+    """
+    Remove ring artifact from sinograms using BM3D method.
```

Comment:
```suggestion
    Remove ring artifact from sinograms using BM3D method.
    To activate this functionality `conda install bm3d-streak-removal`
```

Reply:
I add the instruction to the error message.